roles/nginx/tasks/main.yml

```diff
@@ -92,11 +92,20 @@
     - pepyatka
     - nginx
 
-- name: Fix authentication token perfix in public/js/config.js
+- name: Fix auth.cookieDomain in public/js/config.js
   replace:
     dest: /var/www/pepyatka-html/public/js/config.js
-    regexp: "'tokenPrefix':.*"
-    replace: "'tokenPrefix': '{{ pepyatka_auth_token_prefix }}'"
+    regexp: "cookieDomain:.*"
+    replace: "cookieDomain: '{{ pepyatka_auth_cookie_domain }}'"
+  tags:
+    - pepyatka
+    - nginx
+
+- name: Fix auth.tokenPrefix in public/js/config.js
+  replace:
+    dest: /var/www/pepyatka-html/public/js/config.js
+    regexp: "tokenPrefix:.*"
```

Comment:
Regexp will not match, in pepyatka-html/public/js/config.json tokenPrefix is in single quotes.